astlib/loc.mli

```diff
@@ -9,7 +9,10 @@
 (** {1 Type} *)
 
 (** The type equivalence with [Ocaml_common.Location.loc] is expected to stay stable. *)
-type 'a t = private 'a Ocaml_common.Location.loc
+type 'a t = 'a Ocaml_common.Location.loc =
+  { txt : 'a
```

Comment:
Isn't there a risk that this breaks in the future? I strongly doubt `location` or `loc` would change to be honest but wasn't abstracting those types the whole point of this module?

Reply:
Good point. I've made both types completely abstract now.